lib/sycamore/sycamore/tests/unit/data/test_document.py

```diff
@@ -110,6 +110,14 @@ def test_element_typechecking(self):
             Document({"elements": ["abc"]})
         Document({"elements": [{"type": "special"}]})
 
+    def test_elements_does_not_copy(self):
+        d = Document({"elements": [{"type": "b"}, {"type": "a"}]})
+        assert d.elements[0]["type"] == "b"
+        assert d.elements[1]["type"] == "a"
+        d.elements.sort(key=lambda v: v["type"])
+        assert d.elements[0]["type"] == "a"
+        assert d.elements[1]["type"] == "b"
```

Comment:
This tests that d.elements is not returning a copy of the elements.  If it did then this wouldn't work because sort is in place.  See later where I remove a ton of 
```
e = doc.elements # makes a copy
... do stuff with e ...
doc.elements = e # replace the copy
```
Can you suggest a comment to add?